src/etl/sars-cov-2/steps/fetch-positive-case-data-step.ts

```diff
@@ -277,8 +277,8 @@ export const fetchPositiveCaseDataStep = async(
       const split_csv_line = element.split(",");
       const date = split_csv_line[indexOfDateColumn];
       const year = date.split("-")[0]
-      const month = date.split("-")[1]
-      const day = date.split("-")[2]
+      const month = date.split("-")[1].replace(/^0+/, '')
+      const day = date.split("-")[2].replace(/^0+/, '')
```

Comment:
Probably should have made these numbers, I would have dodged this problem. Whoopsies, we'll come back for this later.